### PR TITLE
Fix redis docker image

### PR DESCRIPTION
**Pull request by farzadmf**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by farzadmf** [comment]
Comment

**Comment by bradtopol** [comment]
@farzadmf Thank you for your contribution. Please sign the CLA so we can accept your PR for review.

**Comment by farzadmf** [comment]
@bradtopol I thought I did a few hours ago, but apparently, I was missing things. I signed it now. I don't know if I need to do anything else

File: content/en/examples/pods/config/redis-pod.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   containers:
   - name: redis
-    image: kubernetes/redis:v1
+    image: k8s.gcr.io/redis:v1
```
Comment (DanyC97) [inline content/en/examples/pods/config/redis-pod.yaml]: do we need a specific image or we can use `image: redis:5` like https://github.com/kubernetes/website/pull/14112/files#diff-7cdce5d48ef944b468c25599bab54d57R8 ?

Comment (farzadmf) [reply content/en/examples/pods/config/redis-pod.yaml]: The "official" `redis` image doesn't work with those `volumeMounts` set up, but this specific image is apparently set up to look in those directories

**Comment by tengqm** [comment]
Looks like #14132 is a better fix for the same issue?

**Comment by farzadmf** [comment]
> Looks like #14132 is a better fix for the same issue?

@tengqm you're right, the other is better in the sense that it's not using a custom image and is using the official one.

I'm not sure what to do with this one. Should I close it?

**Comment by tengqm** [comment]
@farzadmf If #14132 is a better fix, we can close this one.

**Comment by farzadmf** [comment]
Closing this one after discussing with @tengqm and in favor of #14132

**Pull request closed without merging**